pkg/controller/selector/selector_test.go

```diff
@@ -74,11 +74,13 @@ var (
 			TotalCapacity:    99999,
 			FreeCapacity:     5000,
 			DockId:           "ccac4f33-e603-425a-8813-371bbe10566e",
-			Extras: model.ExtraSpec{
-				"thin":     true,
-				"dedupe":   false,
-				"compress": false,
-				"diskType": "SSD",
+			Extras: model.StoragePoolExtraSpec{
```

Comment:
Please follow the format of [examples](https://github.com/opensds/opensds/blob/master/examples/driver/lvm.yaml) to better test

Reply:
It has been modified